assets/sass/patterns/organisms/content-header.scss

```diff
@@ -501,7 +501,7 @@ $iconlist: cc oa;
 }
 
 .content-header__author_link_highlight {
-  padding: 0;
+  padding: 12px 0 0;
```

Comment:
Do the right / left / bottom values need to be set to 0? If not, just use `padding-top`.

Reply:
The reason I have coded it in the way is that padding stands out more than just padding-top so if any other developer reads it and needs to add more padding to the left/right or bottom then it's just a case of adding it. 

I find it easier to read.

Reply:
If the code is correct then it makes sense to try to improve the developer experience. Including rules that aren't needed isn't correct though, and can cause problems down the line. Please use `padding-top` here.

Reply:
Just my opinion